press/errors.py

```diff
@@ -12,7 +12,5 @@ def __init__(self, model):
         self.model = model
 
 
-class CollectionChanged(Exception):
-    """Raised when checked out version is older than published version"""
-    def __init__(self, collection):
-        self.collection = collection
+class AttributeDoesNotExist(Exception):
```

Comment:
Btw, I am totally open to suggestions on a better name for this or if this is even necessary.